src/App.css

```diff
@@ -36,3 +36,39 @@
     transform: rotate(360deg);
   }
 }
+.container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  margin-top: 1%;
+}
+.mb-3 input{
+  width: 100%;
+  border-radius: 20px;
+  border: 1px solid #ccc;
+  padding: 10px;
+  margin-bottom: 10px;
+}
+.mb-3 input[text=text]{
```

Comment:
why `mb-3` ? was this already a class from bootstrap? My mind assumes this means `margin bottom` 

You should use a more descriptive class name, if this will be specific to your login then something like `login-input`, or if it will be general to all inputs, `input-style` or something